sass/_vars.scss

```diff
@@ -9,7 +9,10 @@ $color-bgc: #fff;
 // $color-brand--primary: #ffc53d;
 // $color-brand--secondary: #ff9a3d;
 
-$font-family--primary: "Montserrat", sans-serif;
+$component-shadow: #e60001 0px 0px 35px;
+$component-radius: 10px;
+
+$font-family--primary: 'Raleway', sans-serif;
```

Comment:
Czemu zmieniłeś font? W designach jest Montserrat

Reply:
![image](https://user-images.githubusercontent.com/31566345/103909466-04f7cd80-5104-11eb-91da-4cd7b8c1394f.png)
bo mi błąd wyrzuca... po którymś pull developie tak się stało

Reply:
@PiotrWR nie działa tak jak piszesz. Zmieniłem czcionkę na buttonie, który wywoływał błąd, ponieważ odnosił się do zakomentowanej zmiennej i jest ok.

Reply:
Jak działa to najważniejsze.